### ISSUE TITLE
[Bug]: function col_length(unknown, unknown) does not exist, when use tsql  client.

### ISSUE THREAD

**Issue by giorgiobazzo** [babelfish-for-postgresql/babelfish_extensions#1447]
### What happened?

I'm testing the compatibility of scripts for creating MS SQL Server databases. I could see that the col_length function is not being recognized by Babelfish.

[COL_LENGTH (Transact-SQL)](https://learn.microsoft.com/pt-br/sql/t-sql/functions/col-length-transact-sql?view=sql-server-ver16)

To reproduce the error I'm getting:

```sh
$ tsql -H localhost -U babelfish_user -p 1433  -P 12345678 -D master
1> SELECT COL_LENGTH('mytable','mycolumn')
2> GO
...
Msg 33557097 (severity 16, state 1) from BABELFISH Line 1:
        "function col_length(unknown, unknown) does not exist"

```

### Version

BABEL_3_X_DEV (Default)

### Extension

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by robverschoor** [comment]
Indeed, **`COL_LENGTH()`** is not currently supported by Babelfish. The Babelfish Compass tool would also tell you that if you ran your SQL code through it.
This is unrelated to the client being used.

**Comment by tanscorpio7** [comment]
`COL_LENGTH` is now supported in babelfish https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1906
